roads.mss

```diff
@@ -3049,6 +3060,7 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       text-face-name: @book-fonts;
       text-vertical-alignment: middle;
       text-dy: 7;
+      text-repeat-distance: @text-repeat-distance;
```

Comment:
This code is not working properly in my tests - see my previous comment on `text-repeat-distance` values.